plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1955,13 +1955,13 @@ read_only::get_producer_schedule_result read_only::get_producer_schedule( const
 
 template<typename Api>
 struct resolver_factory {
-   static auto make(const Api* api, const fc::microseconds& max_serialization_time) {
-      return [api, max_serialization_time](const account_name &name) -> optional<abi_serializer> {
+   static auto make(const Api* api, abi_serializer::yield_function_t yield) {
+      return [api, yield{std::move(yield)}](const account_name &name) -> optional<abi_serializer> {
          const auto* accnt = api->db.db().template find<account_object, by_name>(name);
          if (accnt != nullptr) {
             abi_def abi;
             if (abi_serializer::to_abi(accnt->abi, abi)) {
-               return abi_serializer(abi, max_serialization_time);
+               return abi_serializer(abi, yield);
             }
          }
```

Comment:
what should the handling  if accnt == nullptr?

Reply:
If no account then empty optional is returned. There can be no abi if no account.